src/ui/molecules/footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Typography } from 'ui/atoms/typography/Typography'
+import type { I18nResource } from '../../../i18n/utils'
+import { loadTranslations } from '../../../i18n/utils'
+import { useTranslation } from '../../../hook/useTranslation'
+import footer_en from './i18n/footer_en.json'
+import footer_fr from './i18n/footer_fr.json'
+import './footer.scss'
+
+const translationsToLoad: I18nResource[] = [
+  { lng: 'EN', namespace: 'footer', resource: footer_en },
```

Comment:
you have to write the `lng` values in lowercase in order to get your translations files loaded correctly => the displayed translation is not working for now.